### ISSUE TITLE
Error value of config('auth.defaults.guard') when you're logged with another guard.

### ISSUE THREAD

**Issue by willypuzzle** [laravel/framework#22096]
- Laravel Version: 5.5.21
- PHP Version: 7.1.11
- Database Driver & Version:  mysql 7.1.11

### Description:
When I'm logged with another guard than the default guard and I try to get the default guard value with config('auth.defaults.guard') it returns the current guard value (the guard which I'm currently logged in) rather than the real default guard value.

For example, if I have two guards 'web' and 'admin' and the default guard is 'web' and I'm logged with 'admin' when I make config('auth.defaults.guard') it returns 'admin'.

### Steps To Reproduce:
Try to get the default guard value with config function when you're logged with another guard.

**Comment by themsaid** [comment]
Yes laravel switched the default driver used to the current used guard.

**Comment by willypuzzle** [comment]
I think that when I call the config function it returns the config value.

**Comment by GabrielaSaldana** [comment]
Hello, I don't much about guards but I have followed this steps for a web and admin guards and it works perfectly for me, maybe you are missing the configuration of some Laravel Files, checkout this video : https://www.youtube.com/watch?v=iKRLrJXNN4M

**Comment by willypuzzle** [comment]
How Do I suppose to get the default guard if I'm logged with another guard?

**Comment by MladenJanjetovic** [comment]
> This is expected behavior. Laravel will switch the default guard to the current one used.

I don't see how this is expected behaviour. It's confusing.
Default should be default auth guard as per config file, no matter what is the current ACTIVE guard on the route.

**Comment by DevBodin** [comment]
Definitely going to need a better explanation here. Honestly baffled it's been closed.

1. "Default" does not mean "current," 
2. If you manually set a config value, that's the value expect to be returned. 
3. This very predictably leads to the obvious question: "How do I find the **actual** default value?"
